samples/requirements.txt

```
@@ -1,4 +1,4 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
-optimum[openvino]==1.20.0
+optimum[openvino]>=1.20.0
```

```suggestion
optimum[openvino]==1.20.0
```
It's intentionally frozen to ensure reproducibility and updated by dependabot: https://github.com/openvinotoolkit/openvino.genai/blob/master/.github/dependabot.yml. Please revert.

---

It's up to date because dependabot updates once in a week.

---

I reverted it, but I don't think that works as expected. 1.20.0 was released May 29. There have been three releases since that, last one 1.21.2 two weeks ago https://pypi.org/project/optimum/#history .

---

Thank you for the report. Fix: https://github.com/openvinotoolkit/openvino.genai/pull/653